app/src/main/java/fr/iut_rodez/pathpilot_android_client/home/clients/ClientService.java

```diff
@@ -0,0 +1,84 @@
+package fr.iut_rodez.pathpilot_android_client.home.clients;
+
+import static fr.iut_rodez.pathpilot_android_client.util.Network.getRequestQueue;
+import static fr.iut_rodez.pathpilot_android_client.util.VolleyErrorHandler.handleError;
+
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.util.Log;
+import android.widget.ListView;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.JsonObjectRequest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import fr.iut_rodez.pathpilot_android_client.BuildConfig;
+import fr.iut_rodez.pathpilot_android_client.home.Home;
+import fr.iut_rodez.pathpilot_android_client.model.Client;
+import fr.iut_rodez.pathpilot_android_client.model.Client.ClientArrayAdapter;
+
+public class ClientService {
+
+    public static final String API_BASE_URL = BuildConfig.API_BASE_URL + "api/clients";
+    private static final String TAG = ClientService.class.getSimpleName();
+
+    private static RequestQueue requestQueue;
+
+    public static void getClients(Context context, ListView listClientsView) {
```

Comment:
Add singature

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994

Reply:
not on the method, there is supposed to be a signature there

Reply:
Really fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/1828c9ee2511dbcbee079ec147a547bac356039b